### PR TITLE
feat(module:modal): draggable

### PR DESCRIPTION
**Pull request by ParsaArvanehPA**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [✔] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [✔] Tests for the changes have been added (for bug fixes / features)
- [✔] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[✔] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
Ng-zorro's Modal component lacks the [draggable](https://ant.design/components/modal#components-modal-demo-modal-render) feature that Ant designs' modal provides.

## What is the new behavior?
Now supports draggable modals

## Does this PR introduce a breaking change?
```
[ ] Yes
[✔] No
```

### CODE REVIEW

**Comment by lppedd** [comment]
Btw, someone else had attempted at this a while back: https://github.com/NG-ZORRO/ng-zorro-antd/pull/7852

Hopefully this one gets merged.

**Comment by lppedd** [comment]
@ParsaArvanehPA does it make sense for the automation to still assign @hsuanxyz as reviewer?

Seems like he's not involved anymore.

**Comment by ParsaArvanehPA** [comment]
> @ParsaArvanehPA does it make sense for the automation to still assign @hsuanxyz as reviewer?
> 
> Seems like he's not involved anymore.

Assigneess are automatically assigned based on this [file](https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/.github/CODEOWNERS#L13) (I think).

I don't know if they are still involved or not. But other collabrators will check this PR nonetheless.

File: components/modal/modal-container.component.ts [hunk 1]
```diff
@@ -49,7 +53,7 @@ import { ModalOptions } from './modal-types';
           <button nz-modal-close (click)="onCloseClick()"></button>
         }
         @if (config.nzTitle) {
-          <div nz-modal-title></div>
+          <div nz-modal-title cdkDragHandle [style.cursor]="config.nzDraggable ? 'move' : 'default'"></div>
```
Comment (HyperLife1119) [inline components/modal/modal-container.component.ts:56]: Change `default` to `auto`.

**Comment by ParsaArvanehPA** [comment]
@HyperLife1119 
Thank you for checking my PR; changes are done.

**Review by HyperLife1119 — APPROVED** [review]
LGTM

**Comment by lppedd** [comment]
Thanks guys for the fast merge, appreciate!

**Pull request merged**